client/service/src/task_manager/mod.rs

```diff
@@ -63,10 +83,13 @@ impl SpawnTaskHandle {
 	pub fn spawn(
 		&self,
 		name: &'static str,
-		group: Option<&'static str>,
+		group: impl Into<GroupName>,
```

Comment:
the docs need to be updated

Reply:
Alright. Will update the docs

Reply:
I have updated the docs now
can you take a look at the PR again
@ordian